lib/appsignal/transaction.rb

```diff
@@ -33,68 +34,76 @@ def complete_current!
       end
     end
 
-    attr_reader :request_id, :transaction_index, :process_action_event, :action, :exception,
-                :env, :fullpath, :time, :tags, :kind, :queue_start, :paused, :root_event_payload
+    attr_reader :transaction_index, :transaction_id, :namespace, :request, :paused, :tags, :options
 
-    def initialize(request_id, env)
-      @root_event_payload = nil
-      @request_id = request_id
-      @process_action_event = nil
-      @exception = nil
-      @env = env
-      @tags = {}
+    def initialize(transaction_id, namespace, request, options)
+      @transaction_id = transaction_id
+      @namespace = namespace
+      @request = request
       @paused = false
-      @queue_start = -1
-      @transaction_index = Appsignal::Extension.start_transaction(@request_id)
+      @tags = {}
+
+      @options = options
+      @options[:params_method] ||= :params
+
+      @transaction_index = Appsignal::Extension.start_transaction(@transaction_id, @namespace)
     end
 
-    def sanitized_environment
-      @sanitized_environment ||= {}
+    def pause!
```

Comment:
Is pausing still possible? if not we should remove it or at least log a deprication message.